### PR TITLE
[thread-cert] use the same mesh local prefix as Thread harness

### PR DESCRIPTION
**Pull request by bukepo**

### CODE REVIEW

**Comment by librasungirl** [comment]
Travis fails.

You may need explicitly set the mesh local address when bring up nodes. Otherwise, mesh local prefix would be derived from extpanid, i.e. the practical mesh local prefix is still the old one `fdde:ad00:beef::/64`
https://github.com/openthread/openthread/blob/master/src/core/thread/mle.cpp#L163

File: tests/scripts/thread-cert/node.py [hunk 1]
```diff
@@ -1004,6 +1010,11 @@ def set_active_dataset(
             self.send_command(cmd)
             self._expect('Done')
 
+        # Set the meshlocal prefix in config.py
+        self.send_command('dataset meshlocalprefix %s' %
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/node.py:1014]: Add as an argument `mesh_local_prefix=config.MESH_LOCAL_PREFIX` ?

File: tests/scripts/thread-cert/node.py [hunk 2]
```diff
@@ -1033,6 +1044,11 @@ def set_pending_dataset(self,
...
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/node.py:1048]: Add as an argument: `mesh_local_prefix=config.MESH_LOCAL_PREFIX` ?

File: tests/scripts/thread-cert/node.py (hunk 1 above)
Comment (bukepo) [reply tests/scripts/thread-cert/node.py:1014]: I tend not to extend the API until we really need it. Thoughts?

**Comment by bukepo** [comment]
ping~

**Pull request merged**